plugins/aggregators/basicstats/basicstats.go

```diff
@@ -114,23 +127,101 @@ func (m *BasicStats) Add(in telegraf.Metric) {
 }
 
 func (m *BasicStats) Push(acc telegraf.Accumulator) {
```

Comment:
For the first pass, only using the configuration to control which stats are pushed. I'm not trying to optimize the computation of each stat as they are added.

The balance between checking the configuration per calculation seems like a micro optimization we can make later.